implementations.md

```diff
@@ -248,8 +249,8 @@ _None currently support draft-06 or later._
 #### Testing
 
 -   Python
-    -   [hypo\_schema](https://github.com/mlakewood/hypo_schema) (BSD-2-Clause) Creates generators for Hypothesis from JSON Schema
```

Comment:
@Zac-HD do you have any connection to this project?  Yours is obviously more active but we rarely entirely drop projects from the list.

Reply:
No direct connection, but discovering that it's not compatible with recent versions of Hypothesis is what inspired me to start hypothesis-jsonschema

Happy to leave the link if you'd prefer.

Reply:
Let's leave it for now, we'll probably audit the list when we publish the next draft in a month or so anyway, and I can follow up then.

Reply:
@handrews - Done!  Ready to merge?